scipy/optimize/minpack.py

```diff
@@ -751,6 +751,12 @@ def curve_fit(f, xdata, ydata, p0=None, sigma=None, absolute_sigma=False,
     elif jac is None and method != 'lm':
         jac = '2-point'
 
+    if 'args' in kwargs:
+        # The specification for the model function `f` does not support
+        # additional arguments. Refer to the `curve_fit` docstring for
+        # acceptable call signatures of `f`.
+        raise ValueError("'args' is not an acceptable keyword argument.")
```

Comment:
minor: "acceptable" sounds a little odd to me, how about "a supported keyword argument"